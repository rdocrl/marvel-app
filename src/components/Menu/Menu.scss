.navbar {
  background: rgb(20, 20, 20);
  padding: 0.5rem 1rem;
  z-index: 1;

  &__content {
    display: flex;
    justify-content: space-between;
    max-width: 1200px;
    margin: 0 auto;
    padding-left: 1rem;
    padding-right: 1rem;
  }
  &__items {
    display: flex;
    align-items: center;
    max-width: 1200px;
    gap: 1.5rem;
  }

  &__item {
    color: white;
    text-decoration: none;
  }
}

@media screen and (max-width: 480px) {
  .navbar {
    padding: 0.5rem 0;
   &__items {
     gap: 0.5rem;
   }
  }
}