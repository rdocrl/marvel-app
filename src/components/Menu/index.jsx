import { Link } from 'react-router-dom';
import logo from '../../assets/logo.svg';
import './Menu.scss';

const NavBar = () => {
  return (
    <nav className="navbar">
      <div className="navbar__content">
        <img src={logo} alt="Marvel" width={100} />
        <div className="navbar__items">
          <Link to="/comics" className="navbar__item">
            Comics
          </Link>
          <Link to="/characters" className="navbar__item">
            Characters
          </Link>
          <Link to="/favorites" className="navbar__item">
            Favorites
          </Link>
        </div>
      </div>
    </nav>
  );
};

export default NavBar;
