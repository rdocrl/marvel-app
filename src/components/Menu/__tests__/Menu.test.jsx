import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Menu from '../';

describe('<Menu /> tests', () => {
  describe('Snapshots', () => {
    it('Should render', () => {
      const { asFragment } = render(
        <MemoryRouter>
          <Menu />
        </MemoryRouter>
      );
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
