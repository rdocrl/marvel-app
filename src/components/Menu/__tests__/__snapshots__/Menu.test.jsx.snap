// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Menu /> tests Snapshots Should render 1`] = `
<DocumentFragment>
  <nav
    class="navbar"
  >
    <div
      class="navbar__content"
    >
      <img
        alt="Marvel"
        src="logo.svg"
        width="100"
      />
      <div
        class="navbar__items"
      >
        <a
          class="navbar__item"
          href="/comics"
        >
          Comics
        </a>
        <a
          class="navbar__item"
          href="/characters"
        >
          Characters
        </a>
        <a
          class="navbar__item"
          href="/favorites"
        >
          Favorites
        </a>
      </div>
    </div>
  </nav>
</DocumentFragment>
`;
